src/main/java/io/cryostat/net/web/http/RequestHandler.java

```diff
@@ -88,6 +88,10 @@ default List<HttpMimeType> consumes() {
         return List.of();
     }
 
+    default boolean rawJson() {
```

Comment:
Hmm, I wonder if there's another way to do this without introducing this property. I'm thinking maybe adding a new `HttpMimeType` enum value like `JSON_RAW("application/json")`? I think this should still fall into the existing content-negotiation logic just fine, but it would step past the check in the abstract request handler that wraps JSON responses in the API V2 format and fall into just dumping the body as a string to the response body.

Reply:
Makes sense, and that is better than what I did here just putting a new property on every `RequestHandler` implementation. I'll try it out.

Reply:
You could also maybe do a check within the existing `JSON:` switch case and check if the response object body is a String already. If not then do routine of wrapping it up and `gson.toJson()`ing that into the actual webserver response, otherwise if it is a String then just dump it to the webserver response. This way might break things though, not sure.

Reply:
Probably shouldn't break anything because the v2 `[Target]ReportGetHandlers` `IntermediateResponse`s actually return `Path`s, and for the `writeResponse` to go down this logic branch, the endpoint would have to have a `IntermediateResponse<Path>` and have to produce `JSON`, and as far as I know, these are the only ones that do that.

Reply:
Oh, never mind. It does break `CertificatePostHandler`. I will just do the your first suggestion.